README.md

```diff
@@ -60,6 +60,14 @@ Go to the url and give authorization. Copy the auth code.
   ```
   $ node generate-refresh-token.js --authCode <auth code>
   ```
+  
+  If you are using this package as a dependency and local/application-creds lives in your project you can pass the config path as an arguments as follow:
+  
+  ```
+    node node_modules/node-google-dfp-wrapper/generate-authentication-url.js --config $(pwd)'/local/application-creds'
+  node node_modules/node-google-dfp-wrapper/generate-refresh-token.js --config $(pwd)'/local/application-creds' --authCode <auth code>
```

Comment:
To be consistent with the rest of the doc, would you mind unindenting the top command, and adding `$` to the front of the commands?